server/fishtest/schemas.py

```diff
@@ -640,7 +640,7 @@ def flags_must_match(run):
 # about non-validation of runs created with the prior
 # schema.
 
-RUN_VERSION = 4
+RUN_VERSION = 5
```

Comment:
#2139 already raised the version to 5, if I'm not wrong in this PR should be set to 6

https://github.com/official-stockfish/fishtest/blob/e05e6b0f55456422d1e8d48f547a014cb55b47cf/server/fishtest/schemas.py#L643